frontend/src/main/java/org/abs_models/frontend/typechecker/locationtypes/Benchmark.java

```diff
@@ -0,0 +1,86 @@
+package org.abs_models.frontend.typechecker.locationtypes;
```

Comment:
This file looks a bit like it's not necessary after the merge?  We should remove the old location type system since it bit-rotted -- any dead code will lead to maintainer confusion down the road.

Reply:
Yes, that class was just to compare the systems. It is now removed.